Library/Formula/fzf.rb

```diff
@@ -2,13 +2,16 @@
 
 class Fzf < Formula
   homepage "https://github.com/junegunn/fzf"
-  url "https://github.com/junegunn/fzf/archive/0.8.9.tar.gz"
-  sha1 "efbe5955aa09e9d94f304c6177e55060b1926e28"
+  url "https://github.com/junegunn/fzf-bin/releases/download/0.9.0/fzf-0.9.0-homebrew.tgz"
```

Comment:
Updated the formula. It seems to work but I'm not sure if I'm doing the right way. Please let me know if there's something more to be done.